src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
+
+follower = Follower()
```

Comment:
The only time we should have global scope is for `if __name__ == '__main__'`. I'm still not sure what the point of this script is, but if this is how you meant to keep it, you should declare follower in main and pass it to seek. Again, I don't think that's how this script should be.